### ISSUE TITLE
[Enhancement] Add the ability to set the amount of ammunition (volleys) in the weapon

### ISSUE THREAD

**Issue by OtakuDeith** [PavelZinchenko/event-horizon-main#201]
Предлагаю добавить возможность устанавливать количество боеприпасов (залпов) в веапоне.
Например после того как ракетная установка сделает 10 залпов, она перестаёт работать до отступления или выхода из боя

Add the ability to set the amount of ammunition (volleys) in the weapon.
For example, after the rocket launcher makes 10 volleys, it stops working before retreating or exiting the battle

**Comment by ViviX12** [comment]
Better suggestion would be to  give controllable burst values, where you can make custom machinegun type weapon, with time between shots being set by modder, as currently only magazine size and reload between magazines are customizable.
Your suggestion is extremely specific, solution above would give you ability to do that while being a lot more versatile.

**Comment by OtakuDeith** [comment]
> Better suggestion would be to give controllable burst values, where you can make custom machinegun type weapon, with time between shots being set by modder, as currently only magazine size and reload between magazines are customizable. Your suggestion is extremely specific, solution above would give you ability to do that while being a lot more versatile.

I don't think that the number of volleys for the multishot type can be limited in this way.

**Comment by ViviX12** [comment]
> > Better suggestion would be to give controllable burst values, where you can make custom machinegun type weapon, with time between shots being set by modder, as currently only magazine size and reload between magazines are customizable. Your suggestion is extremely specific, solution above would give you ability to do that while being a lot more versatile.
> 
> I don't think that the number of volleys for the multishot type can be limited in this way.

Making those be independent settings can be another part of the proposal.

**Comment by OtakuDeith** [comment]
> > > Better suggestion would be to give controllable burst values, where you can make custom machinegun type weapon, with time between shots being set by modder, as currently only magazine size and reload between magazines are customizable. Your suggestion is extremely specific, solution above would give you ability to do that while being a lot more versatile.
> > 
> > 
> > I don't think that the number of volleys for the multishot type can be limited in this way.
> 
> Making those be independent settings can be another part of the proposal.

agree

**Comment by VladSuhanOFF** [comment]
Ammunition for missiles and torpedoes would not hurt in the game.....
You can even introduce a new device, the projectile factory is called. If it is on the ship, then the ammunition is not limited, if this device is not present, the ammunition is limited!
